src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -555,7 +556,7 @@ export default class ExtraproRegistration extends Mixins(DateMixin) {
       this.isBusinessActive &&
       !!this.business.previousJurisdiction &&
       !!this.business.prevIncorporationDate &&
-      (!this.isContinuationInAffidavitRequired || !!this.business.affidavitFileKey) &&
+      (!this.isContinuationInAffidavitRequired || this.affidavitValid) &&
```

Comment:
It's better for the sub-component to provide its validity instead of computing it here.